contracts/communityFee/OPFCommunityFeeCollector.sol

```diff
@@ -28,7 +28,6 @@ contract OPFCommunityFeeCollector is Ownable {
         address payable newCollector,
         address OPFOwnerAddress
     ) 
-        public
```

Comment:
Removing "public" is a change to the interface and behavior of this smart contract, of course. Yet it doesn't have an interface to adhere to. Therefore why is the change in this PR? Shouldn't it be elsewhere? (And explained why the change is made)

Reply:
Thanks. I wasn't up to speed on this solidity update. Sounds good.